scripts/clowder/quay.sh

```diff
@@ -7,16 +7,56 @@ if [[ -z $QUAY_ROOT ]]; then
   exit
 fi
 
+if [[ "$(uname)" == "Darwin" ]]; then
+  MAC_OS=true
+else
+  MAC_OS=false
+fi
+
+PODMANAGER="podman"
+
+if [[ -n $2 ]]; then
+  PODMANAGER=$2
+
+  if [[ $2 != "docker" ]] && [[ $2 != "podman" ]]; then
+    echo "Second parameter have to be podman or docker."
+    exit 2
+  fi
+fi
+
+echo "Used pod manager: $PODMANAGER"
+
+CACHE=true
+
+if [[ $3 == "no-cache" ]]; then
```

Comment:
the same here, consider either named argument like `--no-cache=true` or env variable

Reply:
👍